javadocs are failing

I am trying to run gradle publish (which runs every target and javadoc is one of them) but gradle is failing for javadoc:

$./gradlew marklogic-rdf4j:publish
:marklogic-rdf4j:generatePomFileForMainJavaPublication
:marklogic-rdf4j:compileJava UP-TO-DATE
:marklogic-rdf4j:processResources UP-TO-DATE
:marklogic-rdf4j:classes UP-TO-DATE
:marklogic-rdf4j:jar UP-TO-DATE
:marklogic-rdf4j:javadoc
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:167: warning: no description for @throws
	 * @throws MarkLogicRdf4jException
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:188: warning: no description for @param
	 * @param f
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:197: warning: no description for @param
	 * @param queryString
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:198: warning: no description for @param
	 * @param bindings
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:199: warning: no description for @param
	 * @param start
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:200: warning: no description for @param
	 * @param pageLength
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:201: warning: no description for @param
	 * @param includeInferred
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:202: warning: no description for @param
	 * @param baseURI
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:203: warning: no description for @return
	 * @return
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:204: warning: no description for @throws
	 * @throws RepositoryException
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:205: warning: no description for @throws
      hrows MalformedQueryException
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:206: warning: no description for @throws
	 * @throws UnauthorizedException
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:207: warning: no description for @throws
	 * @throws QueryInterruptedException
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:227: warning: no description for @param
	 * @param queryString
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:228: warning: no description for @param
	 * @param bindings
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:229: warning: no description for @param
	 * @param includeInferred
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:230: warning: no description for @param
	 * @param baseURI
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:231: warning: no description for @return
	 * @return
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:232: warning: no description for @throws
	 * @throws IOException
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:258: warning: no description for @param
	 * @param queryString
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/cl                                                                                           ient/MarkLogicClient.java:259: warning: no description for @param
	 * @param bindings
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:260: warning: no description for @param
	 * @param includeInferred
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:261: warning: no description for @param
	 * @param baseURI
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:262: warning: no description for @return
	 * @return
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:263: warning: no description for @throws
	 * @throws IOException
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:264: warning: no description for @throws
	 * @throws RepositoryException
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:265: warning: no description for @throws
	 * @throws MalformedQueryException
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:266: warning: no description for @throws
	 * @throws UnauthorizedException
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:267: warning: no description for @throws
	 * @throws QueryInterruptedException
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:277: warning: no description for @param
	 * @param queryString
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:278: warning: no description for @param
	 * @param bindings
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:279: warning: no description for @param
	 * @param includeInferred
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:280: warning: no description for @param
	 * @param baseURI
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:281: warning: no description for @throws
	 * @throws IOException
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:282: warning: no description for @throws
	 * @throws RepositoryException
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:283: warning: no description for @throws
	 * @throws MalformedQueryException
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:284: warning: no description for @throws
	 * @throws UnauthorizedException
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:285: warning: no description for @throws
	 * @throws UpdateExecutionException
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:294: warning: no description for @param
	 * @param file
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:295: warning: n                                                  o description for @param
	 * @param baseURI
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:296: warning: no description for @param
	 * @param dataFormat
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:297: warning: no description for @param
	 * @param contexts
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:298: warning: no description for @throws
	 * @throws RDFParseException
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:313: warning: no description for @param
	 * @param in
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:314: warning: no description for @param
	 * @param baseURI
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:315: warning: no description for @param
	 * @param dataFormat
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:316: warning: no description for @param
	 * @param contexts
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:331: warning: no description for @param
	 * @param in
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:332: warning: no description for @param
	 * @param baseURI
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:333                                                              : warning: no description for @param
	 * @param dataFormat
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:334: warning: no description for @param
	 * @param contexts
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:350: warning: no description for @param
	 * @param baseURI
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:351: warning: no description for @param
	 * @param subject
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:352: warning: no description for @param
	 * @param predicate
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:353: warning: no description for @param
	 * @param object
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:354: warning: no description for @param
	 * @param contexts
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:367: warning: no description for @param
	 * @param baseURI
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:368: warning: no description for @param
	 * @param subject
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:369: warning: no description for @param
	 * @param predicate
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClien                                                                        t.java:370: warning: no description for @param
	 * @param object
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:371: warning: no description for @param
	 * @param contexts
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:386: warning: no description for @param
	 * @param contexts
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:403: warning: no description for @throws
	 * @throws MarkLogicTransactionException
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:422: warning: no description for @throws
	 * @throws MarkLogicTransactionException
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:442: warning: no description for @throws
	 * @throws MarkLogicTransactionException
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:463: warning: no @return
	public boolean isActiveTransaction(){
	               ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:474: warning: no description for @throws
	 * @throws MarkLogicTransactionException
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:487: warning: no description for @return
	 * @return
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:496: warning: no description for @param
	 * @param parserConfig
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:505: warning: no description for @param
	 * @param rulesets
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:530: warning: no description for @return
	 * @return
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:556: warning: no description for @param
	 * @param constrainingQueryDefinition
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:565: warning: no description for @return
	 * @return
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:574: warning: no description for @param
	 * @param graphPerms
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:588: warning: no description for @return
	 * @return
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:600: warning: no description for @param
	 * @param command
	   ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:609: warning: no @return
	public MarkLogicClientImpl getClient(){
	                           ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:105: warning: no @param for databaseClient
	public MarkLogicClient(DatabaseClient databaseClient) {
	       ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:96: warning: no @param for host
	public MarkLogicClient(String host, int port, String user, String password, String database, String auth) {
	       ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:96: warning: no @param for port
	public MarkLogicClient(String host, int port, String user, String password, String database, String auth) {
	       ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:96: warning: no @param for user
	public MarkLogicClient(String host, int port, String user, String password, String database, String auth) {
	       ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:96: warning: no @param for password
	public MarkLogicClient(String host, int port, String user, String password, String database, String auth) {
	       ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:96: warning: no @param for database
	public MarkLogicClient(String host, int port, String user, String password, String database, String auth) {
	       ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClient.java:96: warning: no @param for auth
	public MarkLogicClient(String host, int port, String user, String password, String database, String auth) {
	       ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClientImpl.java:130: warning: no description for @param
     * @param queryString
       ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClientImpl.java:131: warning: no description for @param
     * @param bindings
       ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClientImpl.java:132: warning: no description for @param
     * @param start
       ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClientImpl.java:133: warning: no description for @param
     * @param pageLength
       ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClientImpl.java:134: warning: no description for @param
     * @param tx
       ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClientImpl.java:135: warning: no description for @param
     * @param includeInferred
       ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClientImpl.java:136: warning: no description for @param
     * @param baseURI
       ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClientImpl.java:137: warning: no description for @return
     * @return
       ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClientImpl.java:138: warning: no description for @throws
     * @throws JsonProcessingException
       ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClientImpl.java:147: warning: no description for @param
     * @param queryString
       ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClientImpl.java:148: warning: no description for @param
     * @param bindings
       ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClientImpl.java:149: warning: no description for @param
     * @param handle
       ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClientImpl.java:150: warning: no description for @param
     * @param start
       ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClientImpl.java:151: warning: no description for @param
     * @param pageLength
       ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClientImpl.java:152: warning: no description for @param
     * @param tx
       ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClientImpl.java:153: warning: no description for @param
     * @param includeInferred
       ^
/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClientImpl.java:530: error: invalid use of @return
     * @return
       ^
1 error
100 warnings
:marklogic-rdf4j:javadoc FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':marklogic-rdf4j:javadoc'.
> Javadoc generation failed. Generated Javadoc options file (useful for troubleshooting): '/space/builder/builds/linux64/1.1.0/marklogic-rdf4j/marklogic-rdf4j/build/tmp/javadoc/javadoc.options'

--- a/.gitignore
+++ b/.gitignore
@@ -34,3 +34,4 @@ marklogic-rdf4j-functionaltests/out/
 marklogic-rdf4j/out/
 marklogic-rdf4j-performance/out/
 marklogic-rdf4j-examples/out/
+marklogic-rdf4j/releases/

--- a/README.md
+++ b/README.md
@@ -127,24 +127,22 @@ gradle marklogic-rdf4j:javadoc
 
 ```
 
-### Contributing
 
-Everyone is encouraged to 
+### Support
+
+The marklogic-rdf4j project is maintained by MarkLogic Engineering and distributed
+under the [Apache 2.0 license](LICENSE). It is designed for use in production applications
+with MarkLogic Server. Everyone is encouraged to 
 [file bug reports](https://github.com/marklogic/marklogic-rdf4j/labels/Bug), 
 [feature requests](https://github.com/marklogic/marklogic-rdf4j/labels/enhancement), and
 [pull requests](https://github.com/marklogic/marklogic-rdf4j/pulls) through
 GitHub. This input is critical and will be carefully considered, though we
-cannot promise a specific resolution or timeframe for any request.
-
-Learn how to [contribute](CONTRIBUTING.md).
-
-### Support
-
-The marklogic-rdf4j is maintained by MarkLogic Engineering and distributed
-under the Apache 2.0 license. In addition, MarkLogic provides technical support
-for release tags of the MarkLogic RDF4J API to licensed customers under the
-terms outlined in the Support Handbook. For more information or to sign up for
-support, visit [help.marklogic.com](http://help.marklogic.com).
+cannot promise a specific resolution or timeframe for any request. In addition, MarkLogic provides technical support for [release
+tags](https://github.com/marklogic/marklogic-rdf4j/releases) of `marklogic-rdf4j`
+licensed customers under the terms outlined in the [Support
+Handbook](http://www.marklogic.com/files/Mark_Logic_Support_Handbook.pdf). For
+more information or to sign up for support, visit
+[help.marklogic.com](http://help.marklogic.com).
 
 ### License
 

--- a/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClientImpl.java
+++ b/marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/client/MarkLogicClientImpl.java
@@ -527,7 +527,6 @@ public class MarkLogicClientImpl {
     /**
      * Close client.
      *
-     * @return
      */
     public void close() {
         // close MarkLogicClientImpl

